tsconfig.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
For this part, I'll refer you to our typescript master, @Velenir 

From my side, I just checked you have `strict` mode on.

Probably you should also enable ES6 output so we can later benefit from tree shaking.

Reply:
Yes, would be nice to have two bundles: default with `module: commonjs` , and treeshakable with `module: es6`
Take [build script](https://github.com/gnosis/dex-js/blob/master/package.json#L14) as a reference

Another thing to add for treeshaking is [all these fields](https://github.com/gnosis/dex-js/blob/master/package.json#L34-L38)